packages/core-components/src/layout/Sidebar/Page.tsx

```diff
@@ -73,7 +94,19 @@ export function SidebarPage(props: PropsWithChildren<{}>) {
         toggleSidebarPinState,
       }}
     >
-      <div className={classes.root}>{props.children}</div>
+      <PageContext.Provider value={{ content }}>
```

Comment:
Should the entire value (not just `content`, but the entire object around it too) be `useMemo`:ed on `contentRef` so that it doesn't change on rerenders? Maybe it's not a big deal really since this is so far up by the root, but ...